java/README.md

```diff
@@ -10,34 +10,59 @@ The Semantic Kernel for Java is an SDK that implements the key concepts of the S
 to be used in Java applications and services in an idiomatic way, and to be easily integrated with other Java libraries
 and frameworks.
 
-## Get Started
+## Get started
 
 To run the LLM prompts and semantic functions in this kernel, make sure you have
 an [Open AI API Key](https://openai.com/api/)
 or [Azure Open AI service key](https://learn.microsoft.com/azure/cognitive-services/openai/quickstart?pivots=rest-api).
 
 ### Requirements
 
-To build the semantic kernel, you will need to have:
+To build the project, you will need to have:
 
-- **Required**:
-    - [JDK 8](https://microsoft.com/openjdk/) or newer to build the Semantic Kernel library
-    - [JDK 17](https://microsoft.com/openjdk/) or newer to run samples
+- **Required**: 
+  - [JDK 8](https://microsoft.com/openjdk/) or newer to build the Semantic Kernel library
+  - [JDK 17](https://microsoft.com/openjdk/) or newer to run samples
+  - [Apache Maven](https://maven.apache.org/) 3.9.1 or newer
+  - Azure OpenAI SDK for Java _(see below for instructions)_
 
-### Build the semantic kernel
+## Building the Semantic Kernel
+
+The Semantic Kernel is built using [Apache Maven](https://maven.apache.org/). To build it you will first need to have the Azure OpenAI SDK for Java installed in your local Maven repository. At this moment, this SDK must be built locally from the source.
+
+### Build the Azure OpenAI SDK locally
+
+2. Clone the Azure SDK OpenAI repository
+
+       git clone git@github.com:Azure/azure-sdk-for-java.git
+
+3. Switch to the OpenAI feature branch
+
+       cd azure-sdk-for-java
+       git checkout feature/open-ai
+
+4. Build and install the Azure SDK OpenAI in your local Maven Repository
+
+       mvn -pl sdk/openai/azure-ai-openai -am install
+
+### Build the Semantic Kernel
 
 1. Clone this repository
 
         git clone git@github.com:microsoft/semantic-kernel.git
 
-2. Build the semantic kernel
+2. Build the Semantic Kernel
 
         cd semantic-kernel/java
         mvn install
 
-## Using the semantic kernel
+    - _Note: disable the integration tests if the build of fails for lack of an OpenAI key_
```

Comment:
Tests that require a key will not run by default

Reply:
Fixed in ad5ebd4